src/web/styles.scss

```diff
@@ -358,6 +358,15 @@ a,
   padding-left: 15px;
 }
 
+.row {
+  margin-left: 0;
```

Comment:
Fun fact: the reason why margins are currently off is because [the bootstrap `row` component uses negative margins](https://www.codeply.com/go/ZLV89BwRQF/why-does-bootstrap-use-negative-margins%3F).